static/src/javascripts/projects/commercial/modules/third-party-tags.ts

```diff
@@ -84,17 +84,35 @@ const insertScripts = (
 
 const loadOther = (): void => {
 	const advertisingServices: ThirdPartyTag[] = [
+		// google conversion tracking
+		// https://support.google.com/google-ads/answer/6095821
```

Comment:
If this information moves up to `commercial-core` as JSDoc, you can get the info straight from your IDE.

Reply:
That's much better - https://github.com/guardian/commercial-core/pull/479